idseq_dag/steps/run_star.py

```diff
@@ -10,7 +10,7 @@
 import idseq_dag.util.count as count
 import idseq_dag.util.validate_constants as vc
 
-RE_SPLIT = re.compile('[/\t]')
+RE_SPLIT = re.compile('(/[12])?\t')
```

Comment:
@cdebourcy Restricted to `/1\t`, `/2\t` and `\t` as suggested.

Reply:
thx!